multiqc/plots/bargraph.py

```diff
@@ -183,6 +183,14 @@ def plot(data, cats=None, pconfig=None):
         logger.warning(f"Tried to make bar plot, but had no data: {pconfig.get('id')}")
         return '<p class="text-danger">Error - was not able to plot data.</p>'
 
+    # Add colors to the categories if not set. Since the "plot_defaults" scale is
+    # identical to default scale of the Highcharts JS library, this is not strictly
```

Comment:
NB: It is needed even before updating / changing highcharts, as the static plots use matplotlib and not highcharts. So that code needs these colours.